ddtrace/contrib/azure.functions/patch.py

```diff
@@ -0,0 +1,39 @@
+from ddtrace import config
+from ddtrace.internal.schema import schematize_service_name
+import azure.functions
+
+from ...internal.utils.wrappers import unwrap
+from ddtrace.vendor import wrapt
```

Comment:
Try using these wrappers instead.

```suggestion
from ddtrace.internal.wrapping import unwrap
from ddtrace.internal.wrapping import wrap
```